pyhanko_certvalidator/fetchers/common_utils.py

```diff
@@ -27,19 +27,36 @@
 
 ACCEPTABLE_STRICT_CERT_CONTENT_TYPES = (
     'application/pkix-cert', 'application/pkcs7-mime',
-    'application/x-x509-ca-cert'
+    'application/x-x509-ca-cert', 'application/x-pkcs7-certificates'
 )
 
 ACCEPTABLE_CERT_PEM_ALIASES = (
-    'application/x-pem-file', 'text/plain',
+    'application/x-pem-file', 'text/plain', 'application/octet-stream'
 )
 
 
 def unpack_cert_content(response_data: bytes, content_type: Optional[str],
                         url: str, permit_pem: bool):
+    def unpack_der_pkcs7(pkcs7_data: bytes, pkcs7_url: str):
```

Comment:
Minor nit: maybe make this a top-level function with an underscore name prefix? I don't like using nested functions unless lexical closures are somehow necessary.